.goreleaser.yaml

```diff
@@ -16,11 +16,11 @@ before:
     - go generate ./...
 
 builds:
-  - env:
+  - binary: plugin
+    env:
       - CGO_ENABLED=0
     goos:
       - linux
-      - windows
```

Comment:
FWIW I'm happy to be the Windows boy on this one, I have plenty of machines we can use to test this kind of thing. Though generally people don't use `sshd` on Windows (but you actually can now). More likely people/companies might want a similar thing for RDP that I think we should look into at some point.